packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -203,19 +221,31 @@ contract Attestations is
 
     if (accounts[msg.sender].attestationRequestFeeToken != address(0x0)) {
       require(
-        !isAttestationTimeValid(accounts[msg.sender].mostRecentAttestationRequest) ||
+        isAttestationExpired(accounts[msg.sender].mostRecentAttestationRequest) ||
```

Comment:
Do we have a bug here? Imagine a low `attestationExpiryBlocks` that would allow us to pass this check and specify a different fee token, but then we increase it so the attestation request we originally thought was expired is now not expired?

Reply:
I'm not sure what the technical definition of bug is but I would say in general we have some possibly unintended edge cases when changing parameters by governance. It's hard to anticipate and protect against that, in general and in this specific case I think?